zio/parquetio/reader.go

```diff
@@ -360,18 +363,28 @@ func convertListType(els []*parquet.SchemaElement, i int) (int, *parquetColumn,
 }
 
 func (r *Reader) Read() (*zng.Record, error) {
-	if r.record == int(r.pr.GetNumRows()) {
-		return nil, nil
-	}
-	r.record++
+	var err error
+	if r.buf == nil || len(r.buf) == 0 {
```

Comment:
```suggestion
	if len(r.buf) == 0 {
```
Because `len([]interface{}(nil)) == 0`.